rpc/common/moc_common_common.proto

```diff
@@ -358,3 +358,8 @@ enum CloudInitDataSource {
 	NoCloud = 0;
 	Azure = 1;
 }
+
+message ApiVersion {
```

Comment:
Minor: Ideally it should be Major.Minor.Build.Revision.

Reply:
agree, i didnt want to add fields to protobuf that we are not using not.